CHANGELOG.md

```diff
@@ -40,9 +62,7 @@ disable tqdm progress bar terminal output. Defaults to disabled (`False`).
     `AdditiveKernel` and `ProductKernel`
   - `coreax.kernels.scalar_valued.py` holds all currently implemented scalar valued kernels e.g.
     `SquaredExponentialKernel`.
-
-
-
+  - Refactored `coreax.weights.py` to make weight solvers generic on datatype.
```

Comment:
Can use "data type" here (or possibly even "`Data` type" to be clearer); in either case you don't need the cspell addition.